api/envoy/type/tracing.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+package envoy.type;
+
+option java_outer_classname = "TracingProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.type";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Tracing Commons]
+
+// Describes custom tags for the active span.
+message TracingCustomTag {
```

Comment:
Can you put this into a `tracing` namespace and then just call this message `CustomTag`?